g/github.com__prometheus__client_model/github.com__prometheus__client_model_rhel_8.3.sh

```diff
@@ -1,22 +1,23 @@
 # -----------------------------------------------------------------------------
 #
-# Package	: github.com/prometheus/client_model
-# Version	: v0.1.0
-# Source repo	: https://github.com/prometheus/client_model
-# Tested on	: RHEL 8.3
+# Package       : github.com/prometheus/client_model
+# Version       : v0.0.0-20150212101744-fa8ad6fec335
+# Source repo   : https://github.com/prometheus/client_model
+# Tested on     : RHEL 8.3
 # Script License: Apache License, Version 2 or later
-# Maintainer	: BulkPackageSearch Automation <sethp@us.ibm.com>
+# Maintainer    : BulkPackageSearch Automation <sethp@us.ibm.com>, Raju Sah <Raju.Sah#ibm.com>
 #
 # Disclaimer: This script has been tested in root mode on given
 # ==========  platform using the mentioned version of the package.
 #             It may not work as expected with newer versions of the
 #             package and/or distribution. In such case, please
-#             contact "Maintainer" of this script.
+#             contact "Maintainer" of this script. 0a2904e15265
 #
 # ----------------------------------------------------------------------------
 
 PACKAGE_NAME=github.com/prometheus/client_model
-PACKAGE_VERSION=v0.1.0
+PACKAGE_VERSION=v0.0.0-20150212101744-fa8ad6fec335
```

Comment:
Make PACKAGE_VERSION as configurable.

Reply:
removed extra package version which is being commented.